fastlane/README.md

```diff
@@ -35,7 +35,7 @@ Fetches and updates certificates and provisioning profiles for Ad-Hoc distributi
 [bundle exec] fastlane sync_signing_alpha
 ```
 
-Fetches and updates certificates and provisioning profiles for Alpha TestFlight distribution
+
```

Comment:
This looks like unrelated to this PR, but it seems to match the current develop, so I think it only depends on the branch's history and doesn't signal any unexpected side effect.

Reply:
Yes, it was just an automatic update to fastlane's readme that happens automatically after running fastlane. I suppose this update just wasn't committed to develop when I branched out of it.